ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/CommandDeduplication.scala

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.tests
+
+import java.util.UUID
+
+import com.daml.ledger.api.testtool.infrastructure.Allocation._
+//import com.daml.ledger.api.testtool.infrastructure.Assertions.assertGrpcError
+import com.daml.ledger.api.testtool.infrastructure.{LedgerSession, LedgerTestSuite}
+import com.digitalasset.ledger.test_stable.Test.Dummy
+import com.digitalasset.timer.Delayed
+import com.google.protobuf.duration.Duration
+//import io.grpc.Status
+
+import scala.concurrent.duration.DurationInt
+
+final class CommandDeduplication(session: LedgerSession) extends LedgerTestSuite(session) {
+
+  test(
+    "CDSimpleDeduplication",
+    "Deduplicate commands within the TTL window",
+    allocate(SingleParty),
+  ) {
+    case Participants(Participant(ledger, party)) =>
+      val ttlSeconds = 5
+      val ttl = Duration.of(ttlSeconds.toLong, 0)
+      val a = UUID.randomUUID.toString
+      val b = UUID.randomUUID.toString
+
+      for {
+        request <- ledger.submitRequest(party, Dummy(party).create.command)
+        requestA = request.update(_.commands.ttl := ttl, _.commands.commandId := a)
+
+        // Submit command A (first TTL window)
+        _ <- ledger.submit(requestA)
+        _ <- ledger.submit(requestA)
+
+        // Wait until the end of first TTL window
+        _ <- Delayed.by(ttlSeconds.seconds)(())
```

Comment:
```suggestion
        _ <- Delayed.by(ttl)(())
```

Reply:
Doesn't work, it's the wrong kind of Duration (protobuf vs scala)